.github/workflows/packaging.yml

```diff
@@ -146,10 +197,22 @@ jobs:
           echo "schedule: ${{ github.event.schedule }}" >> $GITHUB_STEP_SUMMARY
           echo "ref: ${{ github.ref }}" >> $GITHUB_STEP_SUMMARY
 
+      - uses: actions/download-artifact@v3
```

Comment:
```suggestion
      - uses: actions/download-artifact@v4
```

Reply:
I changed them to v4 but it was breaking and not downloading the files correctly, so I went back to v3.